hack/.golint_failures

```diff
@@ -357,7 +357,6 @@ pkg/security/podsecuritypolicy/user
 pkg/security/podsecuritypolicy/util
 pkg/securitycontext
 pkg/serviceaccount
-pkg/ssh
```

Comment:
You also need to remove pkg/master/tunneler

Reply:
Thanks for reminding, I have modified and committed again.